src/lib.rs

```diff
@@ -156,18 +156,19 @@ pub fn uring_builder() -> io_uring::Builder {
     io_uring::IoUring::builder()
 }
 
-/// Builder API to allow starting the runtime and creating the io_uring driver with non-default
-/// parameters.
+/// Builder API that can create and start the `io_uring` runtime with non-default parameters,
+/// while abstracting away the underlying io_uring crate.
 // #[derive(Clone, Default)]
 pub struct Builder {
     entries: u32,
     urb: io_uring::Builder,
 }
 
-/// Return a Builder to allow setting parameters before calling the start method.
-/// Returns a Builder with our default values, all of which can be replaced with the methods below.
+/// Constructs a [`Builder`] with default settings.
 ///
-/// Refer to Builder::start for an example.
+/// Use this to alter submission and completion queue parameters, and to `start` the io_uring runtime.
```

Comment:
`and to start the io_uring runtime`. It's better than what we had before. But is it accurate? It is used to start the `tokio-uring` runtime isn't it? I like how the emphasis is on getting to `start` at the end.

Reply:
I'll admit to being confused at the moment between our public `start` and our public `block_on`. I could go look at the code, but maybe it says something about our API or about our documentation here. At least the example of start that comes with the `start` method makes it clear although it is verbose.

Reply:
It seems that the "One True" entrypoint of tokio-uring is `Runtime::block_on`.  

`crate::start` just creates a new tokio_uring::runtime::Runtime then calls `block_on`  so it saves 1 function call. 

The special sauce of `Runtime::block_on` is that it creates an instance of the Uring driver,   and plugs it into the context.  After that it seems to be just vanilla Tokio stuff.

Reply:
I'd say that this creates the runtime but doesn't start it.

Reply:
Fixed.